### ISSUE TITLE
Sorry, I can't open the link to download (train + valid + test set, + pre-trained model). I have been researching this direction recently. I need to reproduce your paper. Can you provide me with a download link?

### ISSUE THREAD

**Issue by wangxinyue11** [juliojj/app-real-age#3]
Sorry, I can't open the link to download (train + valid + test set, + pre-trained model). I have been researching this direction recently. I need to reproduce your paper. Can you provide me with a download link?

**Comment by juliojj** [comment]
The link on google drive is working perfectly. You can also download it from the following (temporary) links:

http://158.109.8.102/AppaRealAgeFG19/train.zip
http://158.109.8.102/AppaRealAgeFG19/valid.zip
http://158.109.8.102/AppaRealAgeFG19/test.zip
http://158.109.8.102/AppaRealAgeFG19/vgg16_app-real-age_fg2019_stage_2_st1-lr_0.0001_st2-lr_0.0001.hdf5

**Comment by wangxinyue11** [comment]
thank you for your help!
